src/System.IO.UnmanagedMemoryStream/tests/UmaTests.cs

```diff
@@ -23,6 +23,13 @@ public static void UmaInvalidReadWrite()
                 Assert.Throws<ArgumentOutOfRangeException>(() => uma.Write(capacity, 12345));
                 Assert.Throws<ArgumentException>(() => uma.Write(capacity - 1, 0.123));
 
+                // UMA should throw when reading bad decimal values (some bits of flags are reserved and must be 0)
+                uma.Write(0, 0); // lo
```

Comment:
> they did, actually

That screen shot shows a build, but no tests as part of it. Try cd'ing into the tests directory and doing msbuild /t:rebuildandtest

> If pointer has a small value (including null of course!), 

Yes, and in fact note that the test is using
"FakeSafeBuffer", which IIRC is true to its name (it's fake) and doesn't actually have any memory associated with it. So attempting to write to it will blow up.